mobile/src/main/java/com/wedriveu/mobile/service/scheduling/SchedulingService.java

```diff
@@ -0,0 +1,26 @@
+package com.wedriveu.mobile.service.scheduling;
+
+import com.google.android.gms.location.places.Place;
+import com.wedriveu.mobile.util.location.LocationServiceListener;
+
+/**
+ * This service interacts with the VehicleService Microservice and asks it to find a proper vehicle based on the coordinates
```

Comment:
Too long line, after "asks it to find a proper vehicle" start a new line.
Maybe you didn't notice but, on IntelliJ when you write an interface/class you can see a vertical light-gray thin line on the right. I use that line to know when i'm writing too long lines.

Also when you want to mention a class in docs like "VehicleService" try to use `{@link className}` or `{@linkplain className}`